parlai/agents/bert_ranker/biencoder_ranker.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+from parlai.core.torch_ranker_agent import TorchRankerAgent
+from .bert_dictionary import BertDictionaryAgent
+from .helpers import get_bert_optimizer, BertWrapper, BertModel, \
```

Comment:
nit: i prefer parens instead of explicit linewrap `\`. Your style is your style though.